modules/data-streams/src/main/java/org/elasticsearch/datastreams/DataStreamIndexSettingsProvider.java

```diff
@@ -92,4 +118,54 @@ public Settings getAdditionalIndexSettings(
         return Settings.EMPTY;
     }
 
+    // Find fields in mapping that are of type keyword and time_series_dimension enabled.
```

Comment:
I'd use javadoc for this. It'll show up when you mouseover the method in your IDE which is kind of nice.